test/blackbox-tests/test-cases/melange/dune

```diff
@@ -9,3 +9,7 @@
 (cram
  (deps %{bin:melc_stdlib_prefix})
  (applies_to merlin))
+
+(cram
+ (deps (package odoc))
```

Comment:
@rgrinberg The odoc tests also rely on the odoc binary, so perhaps they should also be changed too.

Reply:
How do you know the tests only rely on the binary?